stock_shipment_management/wizard/shipment_consignee_setter.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Yannick Vaucher
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more description.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+from openerp import models, fields, api
+
+
+class ConsigneeSetter(models.TransientModel):
+    _name = "shipment.consignee.setter"
+    _inherit = "shipment.value.setter"
+
+    consignee_id = fields.Many2one(
+        'res.partner',
+        'Consignee',
+        help="Shipment consignee"
+    )
+
+    @api.multi
+    def set_value(self):
+        """ Changes the Shipment Consignee and update departure and arrival
+        pickings """
+        for setter in self:
```

Comment:
Can `len(self)` be different from `1`? Would it make sense? Consignees could be different in that case. Maybe we could replace the `for` loop with `self.ensure_one()` ?